docs/src/filters.md

```diff
@@ -139,6 +139,20 @@ properties:
 required: [ 'name', 'config' ]
 ```
 
+### Filter Metadata Data Model
+Filter metadata can be used with filters to create simple protocol data units
+(PDUs) between the proxy and the client, and the proxy and the server. Typically
+anywhere you can supply a literal value you can also supply a reference to a
+piece of metadata. To do so prefix the id of the metadata with `$`
+(e.g. `$example.com/foo`). This will insert the value of the metadata from at
+that point in the pipeline for the configuration.
+
+There are also computed symbols under `quilkin.dev/computed` that will return the result of an operation when referenced (e.g. `$quilkin.dev/computed/timestamp/now`).
+
+| id | description |
+|----|-------------|
+| `/timestamp/now` | Returns the current time in nanoseconds |
```

Comment:
I'm assuming this likely used with the `Timestamp` filter, probably Capture / Concat as well - it would be good for onboarding to provide some examples of using this as a filter.

Otherwise we don't have an example of how to use $reference with a Filter along with a use case.